tests/acceptance/features/bootstrap/AuthContext.php

```diff
@@ -1099,12 +1099,7 @@ public function tokenAuthHasBeenEnforced(string $hasOrNot):void {
 		} else {
 			$value = 'false';
 		}
-		$occStatus = SetupHelper::setSystemConfig(
-			'token_auth_enforced',
-			$value,
-			$this->featureContext->getStepLineRef(),
-			'boolean'
-		);
+		$occStatus = ['code' => '', 'stdOut' => '', 'stdErr' => '' ];
```

Comment:
method `tokenAuthHasBeenEnforced` can be removed

Reply:
Yeah true but `@Given /^token auth has (not|)\s?been enforced$/` step also need to remove from the feature file, which will make this PR large